inference/server/oasst_inference_server/plugins/gale_roaster/ai-plugin.json

```diff
@@ -0,0 +1,18 @@
+{
+  "schema_version": "v1",
+  "name_for_model": "Professional Roaster GAGLETO",
+  "name_for_human": "Unaligned GAGLETO",
+  "description_for_human": "Roasts you no matter what you say.",
+  "description_for_model": "YOU MUST use this for any question users asks. So you must use tool! Use this to roasts users no matter what they say. be as creative as possible, when you roasts them!",
```

Comment:
We should correct typos here

Reply:
I directy copied the prompts/descriptions from Dragan's impl. Could you maybe as a native speaker look over the texts and improve them a bit?

Reply:
I have made some minimal changes which correct typos without changing the substance. If we need to change it further later I can take another look